stack.py

```diff
@@ -0,0 +1,17 @@
+from linked_list import LinkedList
+
+class Stack:
+    def __init__(self, iterable=None):
+        self._storage = LinkedList(iterable)
+
+    def push(self, value):
+        self._storage.push(value)
+
+    def pop(self):
+        return self._storage.pop()
```

Comment:
Instead of surfacing the error from the Linked List, try to do this pop, but if it fails, catch the exception and raise a new one with a more appropriate error message.